shared/js/ui/templates/shared/rating-hero.es6.js

```diff
@@ -54,9 +54,13 @@ function siteRatingSubtitle (isCalculating, rating, isWhitelisted) {
             // wrap this in a single root span otherwise bel complains
             return bel`<span>Enhanced from
     <span class="rating__text-only ${rating.before.toLowerCase()}">
-    ${rating.before}</span> to
+    ${rating.before}
+    <span class="icon icon__plus"></span>
+    </span> to
     <span class="rating__text-only ${rating.after.toLowerCase()}">
-    ${rating.after}</span>
+    ${rating.after}
+    <span class="icon icon__plus"></span>
+    </span>
```

Comment:
Have you thought about how this will look when we plug the grades?

E.g. if we wanted to express C+:

- `rating.before` for the CSS class needs to be `c-plus`
- `rating.before` in the text needs to be just `C`

I'm not sure what the best solution is here, just wanted to flag it.

Reply:
Another thing to keep in mind is that the `+` is not expressed in text means it's not there for screen readers. @jagtalon and I have been looking at label solutions for the hero view, though, so maybe it's fine for now.